src/main/java/gregtech/api/unification/ore/OrePrefix.java

```diff
@@ -325,6 +325,41 @@ public class OrePrefix {
     public static final OrePrefix circuit = new OrePrefix("circuit", -1, null, null, ENABLE_UNIFICATION, null);
     public static final OrePrefix component = new OrePrefix("component", -1, null, null, ENABLE_UNIFICATION, null);
 
+    // Nuclear stuff, introduced by Zalgo and Bruberu
+    public static final OrePrefix fuelRod = new OrePrefix("fuelRod", -1, null, MaterialIconType.fuelRod, 0,
+            material -> material.hasProperty(PropertyKey.FISSION_FUEL),
+            mat -> Arrays.asList(I18n.format("metaitem.nuclear.tooltip.radioactive"),
```

Comment:
Pull out the fission property into local variables in a `-> {}` lambda instead of doing it as a one-liner.